demo/tmail/mailetcontainer.xml

```diff
@@ -98,9 +98,10 @@
             </mailet>
 
             <!-- Put the OpenAIMailet after LocalDelivery so the GPT reply would come after the asking question -->
-            <mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@tmail.com" class="com.linagora.tmail.mailet.OpenAIMailet">
+            <mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@tmail.com" class="com.linagora.tmail.mailet.AIBotMailet">
                 <apiKey>demo</apiKey>
                 <gptAddress>gpt@tmail.com</gptAddress>
+                <model>openai,gpt-4o-mini</model>
```

Comment:
We have configuration to structure things, why parsing a configuration field?

I'd prefer :

```
            <mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@tmail.com" class="com.linagora.tmail.mailet.AIBotMailet">
                <apiKey>demo</apiKey>
                <gptAddress>gpt@tmail.com</gptAddress>
                <vendor>openai</vendor>
                <model>gpt-4o-mini</model>
            </mailet>
        </processor>
```

We should drop BTW the vendor part as only one implem is available today.

Reply:
Done.

Reply:
The idea here was to simplify default resolution: what if the user provided a model but not a vendor or the reverse.